packages/rust-bindings/Cargo.toml

```
@@ -20,7 +20,9 @@ serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
 serde-wasm-bindgen = "0.5"
 thiserror = "1.0"
-wasm-bindgen = { version = "0.2", features = ["serde-serialize"] }
+# this is the precise version of wasm-bindgen needed for compatibility with rustc 1.85, wasm-pack@0.13.1, and wasmjs (at the current
+# type of writing)
+wasm-bindgen = { version = "0.2.93", features = ["serde-serialize"] }
```

The exact version is only locked due to it being in `Cargo.lock`.
We should probably lock it here otherwise running `cargo update` would update it.

```suggestion
wasm-bindgen = { version = "=0.2.93", features = ["serde-serialize"] }
```

---

Makes sense, thanks!